scripts/script.js

```diff
@@ -0,0 +1,80 @@
+let listArray = [];
+
+const form = document.querySelector(".form");
+const input = form.querySelector(".form-input");
+const list = document.querySelector(".list");
+
+form.addEventListener("submit", (x) => {
```

Comment:
Karol left us a helpful note to change these to e/event - which have done throughout - thanks! I'll change this one now.

Reply:
Thanks, Georgia!

Reply:
Thank you, yes it should usually be either e or event.